<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_report_type_selection</class>
 <widget class="QDialog" name="page_report_type_selection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>578</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Report Scope</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_tags">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Tags</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_full_plugin">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <italic>false</italic>
               </font>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Full</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_tags">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
               QTableView::item:selected:!active {
                color:white;
                background-color:  #a1c9f7;
                }

                QTableView::indicator:checked {background-color:black;border: 1px solid #d6d6d6;}
                QTableView::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

                QTableView::indicator {
                width: 13px;
                height: 13px;}
                
</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Tags</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Report Type</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_advance_html">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>Advance HTML</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_standard_html">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>Standard HTML</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_pdf">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>PDF</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_csv">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>CSV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_xml">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>XML</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>33</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_tag_export">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>Export Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_censor_skin_tone_image">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
 QCheckBox::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

QCheckBox::indicator {
width: 12px;
height: 12px;
}
</string>
            </property>
            <property name="text">
             <string>Censor SkinTone Tag Images</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>11</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Report Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_global_report_name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="placeholderText">
            <string>Report name</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>85</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Report Path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_save_report_path">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="placeholderText">
            <string>Report path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_report_save_path">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
